Library/Formula/puddletag.rb

```diff
@@ -55,6 +49,9 @@ def install
 
     bin.install Dir[libexec/"bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    system "sh create_macos_app_bundle.sh --name Puddletag --icon puddletag.png --script #{bin}/puddletag"
```

Comment:
We don't really want to create the `.app` bundle to be honest. Certainly not be default at least.